CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @chrisghill @WillBeebe @mclacore @jaketf
```

Comment:
```suggestion
* @coryodaniel @xpositivityx @chrisghill @WillBeebe @mclacore @jaketf
```

Reply:
you can @team-name here but I don't think they support "1 review from team x and one from team y"